dev-packages/e2e-tests/test-applications/create-next-app/tests/server-transactions.test.ts

```diff
@@ -0,0 +1,71 @@
+import { expect, test } from '@playwright/test';
+import { waitForTransaction } from '@sentry-internal/test-utils';
+
+const authToken = process.env.E2E_TEST_AUTH_TOKEN;
+const sentryTestOrgSlug = process.env.E2E_TEST_SENTRY_ORG_SLUG;
+const sentryTestProject = process.env.E2E_TEST_SENTRY_PROJECT;
+const EVENT_POLLING_TIMEOUT = 90_000;
+
+test('Sends server-side transactions to Sentry', async ({ baseURL }) => {
+  const transactionEventPromise = waitForTransaction('create-next-app', transactionEvent => {
+    return (
+      transactionEvent.contexts?.trace?.op === 'http.server' && transactionEvent.transaction === 'GET /api/success'
+    );
+  });
+
+  await fetch(`${baseURL}/api/success`);
+
+  const transactionEvent = await transactionEventPromise;
+
+  expect(transactionEvent).toEqual(
+    expect.objectContaining({
+      transaction: 'GET /api/success',
+      transaction_info: { source: 'route' },
+      type: 'transaction',
+      contexts: expect.objectContaining({
+        trace: {
+          span_id: expect.any(String),
+          trace_id: expect.any(String),
+          // TODO: Why does this have a parent span ID?
+          parent_span_id: expect.any(String),
```

Comment:
@lforst any idea why we have a parent span ID here? Is this "correct"?

Reply:
That's inded weird. Might be a bug? The code for instrumenting this kind of API route is here: https://github.com/getsentry/sentry-javascript/blob/7de5ea40e29125b0092138e419e0a3be20c296d5/packages/nextjs/src/common/wrapApiHandlerWithSentry.ts#L56